Source/SelfService/Web/applications/microservice/microservices/microservicesDataGridColumns.tsx

```diff
@@ -42,13 +42,23 @@ const StatusCell = (params: GridRenderCellParams<any, HealthStatusTableRowProps[
     );
 };
 
-export const microservicesTableColumns: GridColDef[] = [
+export const microservicesDataGridColumns: GridColDef[] = [
     {
         field: 'name',
         headerName: 'Name',
-        minWidth: 270,
+        minWidth: 200,
         flex: 1,
     },
+    {
+        field: 'environment',
+        headerName: 'Environment',
+        minWidth: 170,
+        flex: 1,
+        valueGetter: ({ row }: HealthStatusTableRowProps) => {
+            const environment = row.environment;
+            return environment === 'Prod' ? 'Production' : environment === 'Dev' ? 'Development' : environment;
```

Comment:
We cannot do this, as we do not decide which environment names a user has. Also, it is quite common to have a Test environment as well. For example some customers have Test and Demo, as well as Dev & Prod

Reply:
To be clear, it should be limited on the input, not the output. IE: When the user creates an application. Developers should recognise the env names as it is in the hosted cluster.